CONTRIBUTING.md

```diff
@@ -37,6 +32,25 @@ it in the commit message (`Fixes #42.`).
 
 [mailmap]: https://github.com/git/git/blob/master/Documentation/mailmap.txt
 
+Setup
+-----
+
+1. Fork the repo.
+2. Install dependencies
+  Cram is used for tests:
+
+    pip install cram
```

Comment:
I think if you indent this and L51 below 2 more spaces over, the commands will be formatted.